### ISSUE TITLE
pagination 

### ISSUE THREAD

**Issue by sebbean** [duartejc/mailchimp#9]
```
Mailchimp.Account.get! 
|> Mailchimp.Account.lists!  
|> Enum.find(fn l -> l.id == "5410954e28" end)   
|> Mailchimp.List.members! 
|> Enum.count
```

I only ever get 10 members

**Comment by duartejc** [comment]
I released a new version (0.0.7) fixing this. 
Now you can call **Mailchimp.List.members!**  passing optional parameters in order to paginate.

For example, in order to get more than 10 members, pass %{count: 100} map parameter.
In order to get items from the 10th entry and ignore the first 9 items pass %{offset: 9, count: 100}

> Mailchimp.Account.get! 
> |> Mailchimp.Account.lists!  
> |> Enum.find(fn l -> l.id == "5410954e28" end)   
> |> Mailchimp.List.members!(%{count: 100}) 
> |> Enum.count


Cheers!

**Comment by sebbean** [comment]
thanks!
